### PR TITLE
Modify the ACME renewing logs level

### PR DESCRIPTION
**Pull request by nmengin**
### What does this PR do?

This PR transforms the `DEBUG` logs generated during the ACME certificates renewing into `INFO` logs in the way to get more information about the process without getting all the others `DEBUG` logs.

### Motivation

Related #2158 and slack support about this subject.

### Additional Notes

This PR does not solve the problem but allows getting more information about it.
It seems to be a concurrent access problem.
A test is currently running in the way to repoduce the problem on a long period.

### CODE REVIEW

File: acme/acme.go [hunk 1]
```diff
@@ -448,7 +477,7 @@ func dnsOverrideDelay(delay int) error {
 			return true, nil
 		}
 	} else if delay < 0 {
-		err = fmt.Errorf("Invalid negative DelayDontCheckDNS: %d", delay)
+		err = fmt.Errorf("invalid negative DelayDontCheckDNS: %d", delay)
```
Comment (dtomcej) [inline acme/acme.go]: How come some of these messages start with a capital letter, and others don't?

Comment (ldez) [reply acme/acme.go]: https://github.com/golang/go/wiki/CodeReviewComments#error-strings

File: acme/acme.go [hunk 2]
```diff
@@ -394,51 +394,80 @@ func (a *ACME) retrieveCertificates() {
...
+	}
+	account = object.(*Account)
+	log.Infof("Renewing certificate %+v in data store", certificateResource.Domains)
+	err = account.DomainsCertificate.renewCertificates(renewedACMECert, certificateResource.Domains)
+	if err != nil {
+		return fmt.Errorf("error renewing certificate: %v in datastore", err)
+	}
+	log.Infof("Commit Certificate %+v renewed in data store", certificateResource.Domains)
+	if err = transaction.Commit(account); err != nil {
+		return fmt.Errorf("error Saving ACME account %+v: %s", account, err.Error())
```
Comment (ldez) [inline acme/acme.go]: `fmt.Errorf("error Saving ACME account %+v: %v", account, err)`

File: acme/acme.go [hunk 3]
```diff
@@ -394,51 +394,80 @@ func (a *ACME) retrieveCertificates() {
...
+				ebo.MaxElapsedTime = 60 * time.Second
+				err = backoff.RetryNotify(safe.OperationWithRecover(operation), ebo, notify)
 				if err != nil {
-					log.Errorf("Error renewing certificate: %v", err)
-					continue
-				}
-
-				if err = transaction.Commit(account); err != nil {
-					log.Errorf("Error Saving ACME account %+v: %s", account, err.Error())
+					log.Errorf("datastore cannot sync: %v", err)
```
Comment (ldez) [inline acme/acme.go]: could you starts with a upper-case letter.

File: acme/acme.go [hunk 4]
```diff
@@ -394,51 +394,80 @@ func (a *ACME) retrieveCertificates() {
...
 			if certificateResource.needRenew() {
-				log.Debugf("Renewing certificate %+v", certificateResource.Domains)
-				renewedCert, err := a.client.RenewCertificate(acme.CertificateResource{
-					Domain:        certificateResource.Certificate.Domain,
-					CertURL:       certificateResource.Certificate.CertURL,
-					CertStableURL: certificateResource.Certificate.CertStableURL,
-					PrivateKey:    certificateResource.Certificate.PrivateKey,
-					Certificate:   certificateResource.Certificate.Certificate,
-				}, true, OSCPMustStaple)
+				log.Infof("Renewing certificate %+v from LE", certificateResource.Domains)
```
Comment (ldez) [inline acme/acme.go]: `log.Infof("Renewing certificate from LE: %+v", certificateResource.Domains)`

File: acme/acme.go [hunk 5]
```diff
@@ -394,51 +394,80 @@ func (a *ACME) retrieveCertificates() {
...
+				renewedACMECert, err := a.renewACMECertificate(certificateResource)
 				if err != nil {
-					log.Errorf("Error renewing certificate: %v", err)
+					log.Errorf("Error renewing from LE certificate: %v", err)
```
Comment (ldez) [inline acme/acme.go]: `log.Errorf("Error renewing certificate from LE: %v", err)`

File: acme/acme.go [hunk 6]
```diff
@@ -394,51 +394,80 @@ func (a *ACME) retrieveCertificates() {
...
+		Domain:        certificateResource.Certificate.Domain,
+		CertURL:       certificateResource.Certificate.CertURL,
+		CertStableURL: certificateResource.Certificate.CertStableURL,
+		PrivateKey:    certificateResource.Certificate.PrivateKey,
+		Certificate:   certificateResource.Certificate.Certificate,
+	}, true, OSCPMustStaple)
+	if err != nil {
+		return nil, err
+	}
+	log.Infof("Renewed certificate %+v from  LE", certificateResource.Domains)
```
Comment (ldez) [inline acme/acme.go]: `log.Infof("Renewed certificate from  LE: %+v", certificateResource.Domains)`

File: acme/acme.go [hunk 7]
```diff
@@ -394,51 +394,80 @@ func (a *ACME) retrieveCertificates() {
...
+	}, nil
+}
+
+func (a *ACME) storeRenewedCertificate(account *Account, certificateResource *DomainsCertificate, renewedACMECert *Certificate) error {
+	transaction, object, err := a.store.Begin()
+	if err != nil {
+		return fmt.Errorf("Error during transaction initialization for renewing certificate: %v", err)
+	}
+	account = object.(*Account)
+	log.Infof("Renewing certificate %+v in data store", certificateResource.Domains)
```
Comment (ldez) [inline acme/acme.go]: `log.Infof("Renewing certificate in data store: %+v", certificateResource.Domains)`

File: acme/acme.go [hunk 8]
```diff
@@ -394,51 +394,80 @@ func (a *ACME) retrieveCertificates() {
...
+	err = account.DomainsCertificate.renewCertificates(renewedACMECert, certificateResource.Domains)
+	if err != nil {
+		return fmt.Errorf("error renewing certificate: %v in datastore", err)
```
Comment (ldez) [inline acme/acme.go]: `return fmt.Errorf("error renewing certificate in datastore: %v", err)`

File: acme/acme.go [hunk 9]
```diff
@@ -394,51 +394,80 @@ func (a *ACME) retrieveCertificates() {
...
+	}
+	log.Infof("Commit Certificate %+v renewed in data store", certificateResource.Domains)
+	if err = transaction.Commit(account); err != nil {
+		return fmt.Errorf("error Saving ACME account %+v: %s", account, err.Error())
+	}
+	oldAccount := a.store.Get().(*Account)
+	for _, oldCertificateResource := range oldAccount.DomainsCertificate.Certs {
+		if oldCertificateResource.Domains.Main == certificateResource.Domains.Main && strings.Join(oldCertificateResource.Domains.SANs, ",") == strings.Join(certificateResource.Domains.SANs, ",") && certificateResource.Certificate != renewedACMECert {
+			return fmt.Errorf("renewed certificate %+v not stored", certificateResource.Domains)
```
Comment (ldez) [inline acme/acme.go]: `return fmt.Errorf("renewed certificate not stored: %+v", certificateResource.Domains)`

File: acme/acme.go [hunk 10]
```diff
@@ -394,51 +394,80 @@ func (a *ACME) retrieveCertificates() {
...
+		}
+	}
+	log.Infof("Certificate %+v successfully renewed in data store", certificateResource.Domains)
```
Comment (ldez) [inline acme/acme.go]: `log.Infof("Certificate successfully renewed in data store: %+v", certificateResource.Domains)`

File: acme/acme.go [hunk 11]
```diff
@@ -394,51 +394,80 @@ func (a *ACME) retrieveCertificates() {
...
+		Certificate:   renewedCert.Certificate,
+	}, nil
+}
+
+func (a *ACME) storeRenewedCertificate(account *Account, certificateResource *DomainsCertificate, renewedACMECert *Certificate) error {
+	transaction, object, err := a.store.Begin()
+	if err != nil {
+		return fmt.Errorf("Error during transaction initialization for renewing certificate: %v", err)
+	}
+	account = object.(*Account)
```
Comment (ldez) [inline acme/acme.go]: Could you add blank lines in this function to aerate the code.

**Pull request merged**